src/agents/supervisor/prompts.py

```diff
@@ -1,112 +1,52 @@
 PLANNER_PROMPT = """
-You are a specialized planner for Kyma and Kubernetes queries, including general questions. Your primary task is to create a concise plan that directly reflects the original query without adding extra steps.
+You are a specialized planner for Kyma and Kubernetes queries, including general questions.
 
-Guidelines:
-1. For queries about Kyma or Kubernetes:
-   - Create a plan that directly mirrors the original query points.
-   - Do not add extra steps or elaborate beyond the original questions.
-   - Maintain the original wording and order of the questions.
-   - Mention the resource information like namespace, resource kind, api version and name if provided.
+Sample Queries and Responses:
 
-2. For unrelated queries:
-   - Provide a direct response without a plan/subtasks.
-   - Set the response attribute accordingly.
+- Kyma or Kubernetes related queries:
```

Comment:
I would like to see what planner outputs. We needed these guidelines so that the original queries wordings are kept.
Let's test it together.

Reply:
All guide lines are still in the prompt , https://github.com/kyma-project/kyma-companion/blob/d2c4843099a3eeb9c3b163eea2d5d7e281cf1a15/src/agents/supervisor/prompts.py#L25